apps/tsilhqotin-language-hub/src/Configs/Radio/validateRadioConfig.ts

```diff
@@ -0,0 +1,37 @@
+import { RadioConfig } from './radio.config';
+
+export const isStringWithNonzeroLength = (input: unknown): input is string =>
+    typeof input === 'string' && input.length > 0;
+
+export const validateRadioConfig = (input: unknown): Error[] => {
```

Comment:
Ideally, you should have a test for this validation as well. But I realize that this radio page is a temporary workaround, and is time-sensitive.